### DISCUSSION TITLE
Seperate Tombstones sound from "music" slider.

### DISCUSSION THREAD

**Discussion by nameonascreen** [open-goal/jak-project#3203] [Jak 2]
I'd like to disable music while i'm speedrunning the game, to use my own music, but in doing so I remove the tombstones sound effect. Would be a nice qol improvement if tombstones sound could be moved to the sfx category.

**Comment by Zedb0T** [comment]
Context and issue here if someone works on this -> https://github.com/open-goal/jak-project/issues/2937

The basic idea as a fix would be to catch the sound on the cpp side, and modify the specific sounds there disregarding whatever volume GOAL sent.

**Reply by Zedb0T** [reply]
![image](https://github.com/open-goal/jak-project/assets/89345505/705db45b-b40e-421d-a5d1-14691178df8d)
